scrapers.js

```diff
@@ -1686,6 +1686,41 @@ let scrapers = [
       };
     }
   },
+  {
+    country: 'FRA',
+    url: 'https://raw.githubusercontent.com/opencovid19-fr/data/master/dist/chiffres-cles.csv',
+    scraper: async function() {
+      let data = await fetch.csv(this.url);
+      let states = [];
+      let date = datetime.getYYYYMMDD(new Date());
+      if (process.env['SCRAPE_DATE']) {
+        date = datetime.getYYYYMMDD(new Date(process.env['SCRAPE_DATE']));
+      }
+      for (let row of data) {
+        let granularity = row.granularite !== undefined ? parse.string(row.granularite) : "";
```

Comment:
Looking at the data, `granularite` has the following fields:

- `departement`
- `region`
- `collectivite-outremer`
- `pays`
- `monde`

I would recommend filtering for `region` and `collectivite-outremer`. `departement` would be equivalent to counties in the US, but we would need to associate them to regions first.

Reply:
added "collectivite-outremer" on line 1703